src/lib/services/data-encoder.ts

```diff
@@ -39,6 +39,7 @@ export async function convertPayloadsWithCodec({
   const encoderResponse: Promise<Payloads> = fetch(endpoint + '/decode', {
     headers,
     method: 'POST',
+    credentials: 'include',
```

Comment:
@robholland and/or @feedmeapples You both have a better sense of all of the use cases. Should we do this by default or make it an environment variable.

Reply:
I believe this is fine without configuration. Unlike the security token setup credentials would only be sent to domain names which had set credentials, so this won't increase risk imho.